CHANGELOG.md

```diff
@@ -6,6 +6,30 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 4.2.0
```

Comment:
@MaKleSoft let me know if there's something you'd like to add/remove/change here